source/admin_manual/mailbox_formats/mbox.rst

```diff
@@ -8,14 +8,16 @@ For information on how to configure mbox in Dovecot, see :ref:`mbox_settings`.
 
 .. warning::
 
-  Mbox format is considered deprecated and is maintained primarily for
-  backwards compatibility and utility purposes (specifically for archival
-  purposes, as mbox allows multiple messages to be natively stored in a
-  single file).
+  Mbox format is considered deprecated and nobody should really be using it
+  anymore for daily use.
 
-  mbox is not being maintained for write fixes or general feature or
-  optimization improvements, so it is not advised to use to actively store
-  production data.
+  It still exists mainly exists to read old mail storages and for backwards
```

Comment:
duplicate words: "still exists mainly exists".  Change to "It is maintained to read ..."